aten/src/ATen/native/quantized/cpu/q_adaavgpool.cpp

```diff
@@ -227,11 +229,64 @@ Tensor q_adaptive_avg_pool2d(const Tensor& input, IntArrayRef output_size) {
     return output;
   }
 }
+
+#ifdef USE_PYTORCH_QNNPACK
+Tensor qnnpack_adaptive_avg_pool2d(
+    const at::Tensor& input,
+    IntArrayRef output_size) {
+    std::array<int64_t, 2> kernel_size;
```

Comment:
Does clang-format put these at the same indent?  Weird.

Reply:
My bad. Fixing this. Thanks for catching.